stateproof/builder.go

```diff
@@ -50,15 +170,15 @@ func (spw *Worker) makeBuilderForRound(rnd basics.Round) (builder, error) {
 	}
 
 	lookback := votersRnd.SubSaturate(basics.Round(hdrProto.StateProofVotersLookback))
-	voters, err := l.VotersForStateProof(lookback)
+	voters, err := votersFetcher.VotersForStateProof(lookback)
```

Comment:
interesting, with this new code it appears `votersFetcher.VotersForStateProof` are being called for each round, and before it was only called on SP message.
How's bad it is from the performance standpoint? Can't we batch `VotersForStateProof` somehow? I think we can and the entire LoadTree/TopOnlineAccounts functions might take ranges to benefit for the batching: the DB data are the same for all rounds in the round, and the difference is only in in-memory-deltas, so it could be a single DB lookup and a loop over the rounds range and deltas.

Reply:
probably fine as a subsequent PR